src/handler.rs

```diff
@@ -166,6 +167,16 @@ pub struct Response<I, E> {
     item: ResponseTypeItem<I, E>,
 }
 
+impl<I, E> fmt::Debug for Response<I, E> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        let mut fmt = fmt.debug_struct("Response");
+        match self.item {
+                ResponseTypeItem::Result(_) => fmt.field("item", &"Result(_)".to_string()),
```

Comment:
I think `to_string` here is not necessary?

Reply:
&str has no known size at compile-time.

Reply:
oh damn, I see, it is due to have debug struct helper prints fields... Ok got it